react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -265,10 +265,34 @@ KhMapFlat.parameters = {
     },
 };
 
-// Map3DLayer.
-const map3DLayer = exampleData[0].layers[3];
-export const Map3DLayer = EditDataTemplate.bind({});
-Map3DLayer.args = {
+// MapLayer. This is never Float32 resolution for properties.
```

Comment:
Let's not add this until we are certain that we want to replace the old version.